File: box/src/main/java/de/qabel/box/storage/exceptions/QblStorageException.java
package de.qabel.box.storage.exceptions;

import de.qabel.core.exceptions.QblException;

public class QblStorageException extends QblException {
    public QblStorageException(Throwable e) {
        super(e.getMessage(), e);
    }

    public QblStorageException(String s) {
        super(s);
    }

    public QblStorageException(String s, Exception e) {
        super(s, e);
    }
}

File: box/src/main/java/de/qabel/box/storage/exceptions/QblStorageException.kt
package de.qabel.box.storage.exceptions

import de.qabel.core.exceptions.QblException

open class QblStorageException : QblException {
    constructor(e: Throwable) : super(e.message, e) {
    }

    constructor(s: String) : super(s) {
    }

    constructor(s: String, e: Exception) : super(s, e) {
    }
}
